.eslintrc

```diff
@@ -0,0 +1,33 @@
+{
+    "root": true,
+    "extends": "airbnb-base",
+    "env": {
+      "node": true,
+      "es6": true,
+      "mocha": true
+    },
+    "rules": {
```

Comment:
@samorahno Please, what informed your choice of these rules?

Reply:
@paradisekelechi, thank you for bringing this to my notice. I found the rules on the pivotal tracker story content, I will go over it again and make use of airbnb